x-pack/plugins/ml/public/jobs/new_job/simple/components/enable_model_plot_checkbox/enable_model_plot_checkbox_directive.js

```diff
@@ -61,7 +61,7 @@ module.directive('mlEnableModelPlotCheckbox', function (i18n) {
             if (validationResult.success === true) {
               $scope.formConfig.enableModelPlot = true;
               $scope.ui.cardinalityValidator.status = STATUS.FINISHED;
-
+            } else {
               $scope.ui.cardinalityValidator.message = i18n('xpack.ml.newJob.simple.enableModelPlot.enableModelPlotDescription', {
                 defaultMessage: 'Creating model plots is resource intensive and not recommended' +
                   'where the cardinality of the selected fields is greater than 100. Estimated cardinality' +
```

Comment:
There is missing whitespace after `recommended` and also before `If you enable model`. There is also missing whitespace in the `errorHandler` message further up in the file.

Reply:
Added the missing white space 👌